ipengine_config.py

```diff
@@ -407,10 +393,10 @@
 # Configurable for MPI initialization
 
```

Comment:
To see it in the terminal you need to `print(..., file=sys.__stdout__)` for engines because sys.stdout/err are forwarded over zmq. It's possible startup files aren't run by engines, and/or that's fixed in IPython 3. I'll look into it.